docs/ibexa_cloud/ddev_and_ibexa_cloud.md

```diff
@@ -6,59 +6,58 @@ description: Use DDEV to run an Ibexa Cloud project locally.
 
 Two ways are available to run an [[= product_name_cloud =]] project locally with DDEV:
 
-- [by using the Platform.sh's `ddev-platformsh` add-on](#with-the-ddev-platformsh-add-on)
-- [like other existing project, without this add-on](#without-the-platformsh-add-on).
+- [by using the Platform.sh's `ddev-ibexa-cloud` add-on](#with-the-ddev-ibexa-cloud-add-on)
+- [like other existing project, without this add-on](#without-the-ibexa-cloud-add-on).
 
 !!! note
 
     The following examples use [[[= product_name_cloud =]] CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
 
-## With the `ddev-platformsh` add-on
+## With the `ddev-ibexa-cloud` add-on
 
-To configure the [`ddev/ddev-platformsh` add-on](https://github.com/ddev/ddev-platformsh), you need a [Platform.sh API Token](https://docs.platform.sh/administration/cli/api-tokens.html).
+To configure the [`ddev/ddev-ibexa-cloud` add-on](https://github.com/ddev/ddev-ibexa-cloud), you need a [Platform.sh API Token](https://docs.platform.sh/administration/cli/api-tokens.html).
 
-The `ddev/ddev-platformsh` add-on configures the document root, the PHP version, the database, and the cache pool according to the [[= product_name_cloud =]] configuration.
+The `ddev/ddev-ibexa-cloud` add-on configures the document root, the PHP version, the database, and the cache pool according to the [[= product_name_cloud =]] configuration.
 About the search engine, the add-on can configure Elasticsearch but can't configure Solr.
 If you use Solr on [[= product_name_cloud =]] and want to add it to your DDEV stack, see [Clustering with DDEV and `ibexa/ddev-solr` add-on](clustering_with_ddev.md#solr).
 
-`COMPOSER_AUTH` from Platform.sh can't be used, because JSON commas are incorrectly interpreted by `--web-environment-add`, which sees them as multiple variable separators.
+`env:COMPOSER_AUTH` from Platform.sh can't be used, because JSON commas are incorrectly interpreted by `--web-environment-add`, which sees them as multiple variable separators.
 But the variable must exist for Platform.sh `hooks` scripts to work.
```

Comment:
This seems not true anymore. `hooks`, even the whole `.platform.app.yaml` file, seem unused with [ddev/ddev-ibexa-cloud v0.1.0](https://github.com/ddev/ddev-ibexa-cloud/releases/tag/v0.1.0).